docs/modules/device-lab-management/pages/ios-devices/add-ios-device.adoc

```diff
@@ -0,0 +1,318 @@
+= Add iOS device
+
+== Before you start
+
+* Follow this guide to xref:ios-devices/prepare-ios-device.adoc[prepare the device].
+
+* Find the UDID of the device and note it down.
+
+* Make sure the Mac mini host has the Xcode version that is compatible with the iOS/iPadOS version of the device.
+
+* Export the appropriate xref:ios-devices/generate-an-ios-signing-certificate-and-provisioning-profile.adoc#_generate_a_signing_certificate[signing certificates] (`.p12`) and xref:ios-devices/generate-an-ios-signing-certificate-and-provisioning-profile.adoc#_generate_a_provisioning_profile[provisioning profiles] (`.mobileprovision`) for the UDID of the device.
+
+* Download Apple Worldwide Relations (WWDR) Intermediate certificate file (`AppleWWDRCAG3.cer`) from https://www.apple.com/certificateauthority/AppleWWDRCAG3.cer[this link].
+
+== Import developer certificates and provisioning profiles
+
+Follow each subsection below in order.
+
+=== Import developer certificates
+
+[IMPORTANT]
+These steps require accessing the Mac mini host’s screen and cannot be done via SSH.
+
+Open **Terminal** on the Mac mini host and enter the following command:
+
+----
+sudo security add-certificates -k /Library/Keychains/System.keychain <Path to the cert>/AppleWWDRCAG3.cer
+----
+
+Replace *<Path to the cert>* with the full path to the folder containing the Apple WWDR Intermediate certificate file (`AppleWWDRCAG3.cer`). Enter the administration password if required.
+
+Next, enter the following command, replacing *<Path to the cert>* with the full path to the `.p12` certificate files and *<Name of the cert>* with the filename.
+
+----
+sudo security import <Path to the cert>/<Name of the cert>.p12 -k /Library/Keychains/System.keychain -A
+----
+
+Enter the certificate password. If the certificate has no password, leave the field blank and click **OK**.
+
+image::device-lab-management:device-lab-management-ios-add-ios-certificate-password.PNG[width=300,alt="Enter Password for Kobiton"]
+
+Repeat the above commands for each `.p12` file if there are multiple files.
+
+=== Import provisioning profiles to deviceConnect
+
+Before importing, if your deployment include multiple Mac mini hosts, make sure the Mac mini host has deviceConnect installed by opening the Chrome browser, navigating to the address: *localhost/#/System/IOS*, and logging in.

```

Comment:
Should this be an "Important" section?
I rewrote this, but not sure if it's accurate: "If your deployment includes multiple Mac mini hosts, ensure that deviceShare is installed on the current Mac mini host before proceeding. To verify this, navigate to the deviceShare installation folder and check for the presence of a version number. If no version number is found, it means deviceShare is not installed. In this case, locate another Mac mini host where deviceShare is installed to continue with this section."

Reply:
updated